CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required( VERSION 2.8.7 )
 project(GZDoom)
 
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9")
```

Comment:
Please move this line to the appropriate platform-specific [location](https://github.com/coelckers/gzdoom/blob/af18c78156d7196eaeabd215c190a84c4e74036f/CMakeLists.txt#L291). We don't need such details at the beginning of the main configuration file.